packages/framework-integration-tests/integration/helper/fileHelper.ts

```diff
@@ -1,15 +1,69 @@
-import util = require('util')
-import * as fs from 'fs'
+import {
+  existsSync,
+  mkdirSync,
+  readdirSync,
+  readFileSync,
+  rmdirSync,
+  unlinkSync,
+  writeFileSync,
+  copyFileSync,
+} from 'fs'
+import path = require('path')
 
-const unlink = util.promisify(require('fs').unlink)
-const rmdir = util.promisify(require('fs').rmdir)
+export const loadFixture = (fixturePath: string, replacements?: Array<Array<string>>): string => {
+  const template = readFileContent(`integration/fixtures/${fixturePath}`)
+  return (
+    replacements?.reduce(
+      (prevContents: string, replacement: string[]): string => prevContents.replace(replacement[0], replacement[1]),
+      template
+    ) ?? template
+  )
+}
 
-export const readFileContent = (filePath: string): string => fs.readFileSync(filePath, 'utf-8')
+export const readFileContent = (filePath: string): string => readFileSync(filePath, 'utf-8')
 
-export const writeFileContent = (filePath: string, data: string): void => fs.writeFileSync(filePath, data)
+export const writeFileContent = (filePath: string, data: string): void => writeFileSync(filePath, data)
 
-export const removeFiles = (filePaths: Array<string>): Array<Promise<void>> =>
-  filePaths.map((file: string) => unlink(file))
+export const removeFiles = (filePaths: Array<string>, ignoreErrors = false): void => {
+  filePaths.map((file: string) => {
+    try {
+      unlinkSync(file)
+    } catch (e) {
+      if (!ignoreErrors) throw e
+    }
+  })
+}
 
-export const removeFolders = (paths: Array<string>): Array<Promise<void>> =>
-  paths.map((path: string) => rmdir(path, { recursive: true }))
+export const createFolder = (folder: string): void => {
```

Comment:
Maybe it'd be better to call this `ensureFolderExists`? As it doesn't create it if it exists

Reply:
That makes sense...